crates/matrix-sdk-base/src/store/memory_store.rs

```diff
@@ -351,10 +352,18 @@ impl MemoryStore {
             }
         }
 
+        let make_room_version = |room_id| {
+            self.room_info
+                .get(room_id)
+                .and_then(|info| info.room_version().cloned())
+                .unwrap_or_else(|| {
+                    warn!(%room_id, "Unable to find the room version, assuming version 9");
```

Comment:
Why would we default to v9?

Reply:
this is just a code move from below. I don't know if that is a correct assumption ...

Reply:
Okay, can leave it as-is then, I guess.